debian/control

```diff
@@ -123,6 +123,7 @@ Depends: binutils,
          parted,
          psmisc,
          xfsprogs,
+         python-requests,
```

Comment:
why do you think we should move python-requests from ceph-common to ceph-base ?

Reply:
updated, thanks.